src/you_get/common.py

```diff
@@ -682,19 +690,21 @@ def __init__(self, total_size, total_pieces = 1):
     def update(self):
         self.displayed = True
         bar_size = self.bar_size
-        percent = round(self.received * 100 / self.total_size, 1)
-        if percent >= 100:
+        percent_str = round_str(self.received * 100 / self.total_size, 1)
+        if self.received == self.total_size:
             percent = 100
```

Comment:
There is one more leak issue though: If the size of downloaded pieces already rounds to 100.0% while the download is not finished (`self.received < self.total_size`), the string `100.0%` still emerges.

Example: `you-get --itag=22 https://www.youtube.com/watch?v=5j_lVn8Z9yA`
(Notice that 55836672 / 55844492 => 100.0%)